src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -250,10 +290,31 @@ protected virtual IEnumerable<ServiceBusPlugin> DefineServiceBusPlugins()
         /// <param name="cancellationToken">Indicates that the shutdown process should no longer be graceful.</param>
         public override async Task StopAsync(CancellationToken cancellationToken)
         {
+            if (Settings.Options.IncludeTopicSubscription)
```

Comment:
Creating is one thing, deleting is another.

I'd suggest to leave this out or do this via another config option.